src/NzbDrone.Core/Datastore/Migration/001_initial_setup.cs

```diff
@@ -66,101 +75,253 @@ protected override void MainDbUpgrade()
+                .WithColumn("Name").AsString().NotNullable()
+                .WithColumn("Body").AsString().NotNullable()
+                .WithColumn("Priority").AsInt32().NotNullable()
+                .WithColumn("Status").AsInt32().NotNullable()
+                .WithColumn("QueuedAt").AsDateTime().NotNullable()
+                .WithColumn("StartedAt").AsDateTime().Nullable()
+                .WithColumn("EndedAt").AsDateTime().Nullable()
+                .WithColumn("Duration").AsString().Nullable()
+                .WithColumn("Exception").AsString().Nullable()
+                .WithColumn("Trigger").AsInt32().NotNullable();
+
+            Create.TableForModel("IndexerStatus")
+                .WithColumn("IndexerId").AsInt32().NotNullable().Unique()
+                .WithColumn("InitialFailure").AsDateTime().Nullable()
+                .WithColumn("MostRecentFailure").AsDateTime().Nullable()
+                .WithColumn("EscalationLevel").AsInt32().NotNullable()
+                .WithColumn("DisabledTill").AsDateTime().Nullable()
+                .WithColumn("LastRssSyncReleaseInfo").AsString().Nullable();
+
+            Create.TableForModel("ExtraFiles")
+                .WithColumn("SeriesId").AsInt32().NotNullable()
+                .WithColumn("SeasonNumber").AsInt32().NotNullable()
+                .WithColumn("EpisodeFileId").AsInt32().NotNullable()
+                .WithColumn("RelativePath").AsString().NotNullable()
+                .WithColumn("Extension").AsString().NotNullable()
+                .WithColumn("Added").AsDateTime().NotNullable()
+                .WithColumn("LastUpdated").AsDateTime().NotNullable();
+
+            Create.TableForModel("SubtitleFiles")
+                .WithColumn("SeriesId").AsInt32().NotNullable()
+                .WithColumn("SeasonNumber").AsInt32().NotNullable()
+                .WithColumn("EpisodeFileId").AsInt32().NotNullable()
+                .WithColumn("RelativePath").AsString().NotNullable()
+                .WithColumn("Extension").AsString().NotNullable()
+                .WithColumn("Added").AsDateTime().NotNullable()
+                .WithColumn("LastUpdated").AsDateTime().NotNullable()
+                .WithColumn("Language").AsInt32().NotNullable();
+
+            Create.Index().OnTable("Blacklist").OnColumn("SeriesId");
+
+            Create.Index().OnTable("EpisodeFiles").OnColumn("SeriesId");
+
+            Create.Index().OnTable("Episodes").OnColumn("EpisodeFileId");
+            Create.Index().OnTable("Episodes").OnColumn("SeriesId");
+
+            Create.Index().OnTable("History").OnColumn("EpisodeId");
+            Create.Index().OnTable("History").OnColumn("Date");
+
+            Create.Index().OnTable("Series").OnColumn("Path");
+            Create.Index().OnTable("Series").OnColumn("CleanTitle");
+            Create.Index().OnTable("Series").OnColumn("TvRageId");
+            Create.Index().OnTable("Series").OnColumn("TvMazeId");
```

Comment:
Probably don't need to create these indexes since the tables get removed in early Radarr migrations and the tables are not used

Reply:
Yeah, I'd say drop those then good to go